### PR TITLE
Siemens OOB-20210414

### PR DESCRIPTION
**Pull request by productcert**

### CODE REVIEW

File: 2021/27xxx/CVE-2021-27394.json [hunk 1]
```diff
@@ -1,18 +1,125 @@
 {
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27394.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2021/27xxx/CVE-2021-27394.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2021/27xxx/CVE-2021-27394.json:1]: 'references' is a required property

Comment (cve-team) [inline 2021/27xxx/CVE-2021-27394.json:1]: 'description' is a required property

Comment (cve-team) [inline 2021/27xxx/CVE-2021-27394.json:1]: '5.0' is not one of ['4.0']

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by productcert** [comment]
thank you and have a good weekend

**Pull request merged**